Xamarin.Essentials/Xamarin.Essentials.csproj

```diff
@@ -8,7 +8,7 @@
     <PackageIcon>Assets\xamarin.essentials_128x128.png</PackageIcon>
     <Summary>Xamarin.Essentials: a kit of essential API's for your apps</Summary>
     <PackageTags>xamarin, windows, ios, android, xamarin.forms, watchos, tvos, toolkit, Xamarin.Essentials, kit</PackageTags>
-    <Title>Xamarin.Essentials</Title>
+    <Title>DotNetEssentials</Title>
```

Comment:
@dcghub you changed Title, but I think you have to change PackageId

Reply:
Apparently today it's not being my brightest day...
Should RootNamespace and/or AssemblyName also be changed?

Reply:
nope, we just need to change PackageId for the nuget upload to the proper package

Reply:
please leavee the title as it was

Reply:
This is done too